hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/TypedTable.java

```diff
@@ -456,6 +456,27 @@ public long getEstimatedKeyCount() throws IOException {
     return rawTable.getEstimatedKeyCount();
   }
 
+  /**
+   * Returns the exact key count of column family.
+   *
+   * @return keyCount
+   * @throws IOException
+   */
+  @Override
+  public long getExactKeyCount() throws IOException {
+    if (cache.getCacheType() == CacheType.FULL_CACHE) {
+      return cache.size();
+    }
+    try (TableIterator<byte[], KeyValue<byte[], byte[]>> iterator
+             = rawTable.iterator()) {
+      int keyCount = 0;
+      for (iterator.seekToFirst(); iterator.hasNext(); iterator.next()) {
+        keyCount++;
+      }
+      return keyCount;
+    }
```

Comment:
Please move this to `RDBTable.getExactKeyCount()`, of course omitting `rawTable`, and replace this code in `TypedTable` with:

```suggestion
    return rawTable.getExactKeyCount();
```